source/jormungandr/jormungandr/scenarios/ridesharing/ouestgo.py

```diff
@@ -123,42 +123,19 @@ def _make_response(self, raw_json, request_datetime, from_coord, to_coord, insta
                 res.ridesharing_ad = offer.get('journeys', {}).get('url')
                 res.duration = offer.get('journeys', {}).get('duration')
 
-                # coord of departure on foot to arrive at ride-sharing point
+                # ride-sharing pick up point is the same as departure
                 lat, lon = from_coord.split(',')
-                departure_coord = Coords(lat=lat, lon=lon)
-
-                # ride-sharing pick up coord
-                pickup_lat = float(offer.get('journeys', {}).get('from', {}).get('latitude'))
-                pickup_lon = float(offer.get('journeys', {}).get('from', {}).get('longitude'))
-                pickup_coord = Coords(lat=pickup_lat, lon=pickup_lon)
-
-                res.pickup_place = rsj.Place(addr='', lat=pickup_lat, lon=pickup_lon)
-
+                res.pickup_place = rsj.Place(addr='', lat=float(lat), lon=float(lon))
                 res.origin_pickup_shape = None  # Not specified
-                res.origin_pickup_distance = int(crowfly_distance_between(departure_coord, pickup_coord))
-                # we choose to calculate with speed=1.12 the average speed for a walker
-                res.origin_pickup_duration = get_manhattan_duration(
-                    res.origin_pickup_distance, instance_params.walking_speed
```

Comment:
Seen in sonar: You may remove `instance_params` from the fn.

Reply:
:+1: